### ISSUE TITLE
Gives "Failed - Blocked" message

### ISSUE THREAD

**Issue by walter-schwarz** [JosephusPaye/drag-to-download#2]
This extension used to work fine. But recently I always get a "Failed - Blocked" message when I drag a file link to the desktop.
Tested on several different PCs.
It looks like Chrome changed something in the most recent update.
Can this be fixed?
Thank you!

**Comment by JosephusPaye** [comment]
Yeah, perhaps something has changed. I am able to get it working by disabling and re-enabling the extension, and then re-enabling on the domain.

Could you try that and see if it works? Also, where do you see the "Failed - Blocked" message? Is it in the downloads list at the bottom of Chrome?

It might be worth trying a different site and download link as well.

**Comment by walter-schwarz** [comment]
Yes, I see "Failed - Blocked" message in the downloads bar that appears at the bottom of Chrome.
The problem seems to be with how the Chrome specific "DownloadURL" data transfer type works now. It might do some security check to allow it.
I created an example to illustrate the problem better: [https://codepen.io/wschwarz/pen/poRjEgJ](https://codepen.io/wschwarz/pen/poRjEgJ)

But it is inconsistent. On one of my projects it still works, on others it does not.
To test I added exactly the same code with the same images on 2 different domains, one that works and on another one that does not.
Here is a video of it working: https://user-images.githubusercontent.com/2886164/112658521-c399d300-8e53-11eb-857a-a87c064b163b.mp4
On the other ones it shows the "Failed - Blocked" message like you see in the Codepen example.

The weird thing is that on the domain where it works, if I add that code outside of the app, in a separate file, then it does not work anymore. The only place I could get it to work was inside the app. Maybe some headers that enables it?
But the code is definitely the same, and the elements are not altered by other code. I checked the elements and the attached events with Dev tools and it looks exactly like in the Codepen example. Its all very strange.

**Comment by sanxFlower** [comment]
hello, i had met this problem recently and truly looking forward your reply if you resolved this sucessfully
